composer.json

```diff
@@ -11,9 +11,8 @@
     }],
     "require": {
         "php": "^5.4 || ^7.0",
-        "symfony/css-selector": "2.8.* || ^3.0",
-        "facebook/graph-sdk": "~5.0",
-        "apache/log4php": "2.3.0"
+        "symfony/css-selector": "2.8.*",
```

Comment:
Can we upgrade this to latest one(i guess 4.0.3) if no dependency with this version(2.8.*)?

Reply:
Anyone is able to upgrade it, but we cannot move to 3.2 and up since it is not php 5.4 compatible.
We need to re-think the need of keeping php 5.4 compatibility